script/scp-hosts.sh

```diff
@@ -35,8 +35,8 @@ hostsArr=(${ips//,/ })
 for host in ${hostsArr[@]}
 do
 
-  if ! ssh -p $sshPort $host test -e $installPath; then
-    ssh -p $sshPort $host "sudo mkdir -p $installPath; sudo chown -R $deployUser:$deployUser $installPath"
+  if ! ssh -o StrictHostKeyChecking=no -p $sshPort $host test -e $installPath; then
+    ssh -o StrictHostKeyChecking=no -p $sshPort $host "sudo mkdir -p $installPath; sudo chown -R $deployUser:$deployUser $installPath"
```

Comment:
Would you please add this to the `script/status-all.sh`?

Reply:
> Would you please add this to the `script/status-all.sh`?

Sure.